### PR TITLE
dummy toolchain

### PR DESCRIPTION
**Pull request by wpoely86**

### CODE REVIEW

File: docs/Writing_easyconfig_files.rst [hunk 1]
```diff
@@ -74,7 +74,7 @@ A handful of easyconfig parameters are `mandatory`:
 
 * **name, version**: specify what software (version) to build
 * **homepage, description**: metadata (used for module help)
-* **toolchain**: specifies name and version of compiler toolchain to use
+* **toolchain**: specifies name and version of compiler toolchain to use. A complete list of supported toolchains can be found at :ref:`eb_list_toolchains`
```
Comment (stdweird) [inline docs/Writing_easyconfig_files.rst]: line too long

Comment (wpoely86) [inline docs/Writing_easyconfig_files.rst]: You want to wrap at 72? Or let html handle the wrapping?

File: docs/eb_list_toolchains.rst [hunk 2]
```diff
@@ -12,3 +12,4 @@ The list of toolchains can easily be obtained with::
 
 .. include:: version-specific/eb_list_toolchains.txt
 
+There is one special case: the `dummy` toolchain. This will not load anything and so you will be using whatever compiler can be found in the current path. If you also specify `dummy` as the toolchainversion, no dependencies will be loaded. If you leav the toolchainversion empty, the dependencies will get loaded.
```
Comment (stdweird) [inline docs/eb_list_toolchains.rst]: line way too long
also typo in `leav` (and might look a bit odd, but add examples people can copy/paste?)

**Comment by boegel** [comment]
going in, thanks for initiating this @wpoely86!

**Pull request merged**